starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,66 @@
+body{
+  width: 100%;
+  margin: 0;
+  padding:0;
+  background-color: black;
+  font-family: 'Open Sans', sans-serif;
+}
+
+.top-image{
+  background-image: url("../images/apple-pie.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 700px;
```

Comment:
I think 700px is too big since the page we are cloning has it as a 1/3 of the page, so maybe try adjusting it to 500px or so. Also, if you want the whole image to fit you can use something like:
```
background: url("../images/apple-pie.jpg") no-repeat center center;
height: 500px;
background-size: 100%;
```